test/functional/mempool_accept_wtxid.py

```diff
@@ -4,9 +4,9 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """
 Test mempool acceptance in case of an already known transaction
-with identical non-witness data different witness.
+with identical non-witness data but different witness.
 """
-
+import copy
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), you want a line between standard library and local imports.

Also, as sriramdvt mentioned earlier, a more granular import of just `deepcopy` would be preferred.